From: Michael Catanzaro <mcatanzaro@redhat.com>
Date: Thu, 19 Jan 2023 20:07:40 -0600
Subject: Update for removal of WebKitGTK sandbox API

WebKitGTK 2.39.5 contains an ABI break. The sandbox is now mandatory, so
the API to enable/disable it is gone.

(cherry picked from commit 1a8f9ecfceb2e3106910b7bf7987f0fc75886670)
---
 gnome-initial-setup/gis-driver.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gnome-initial-setup/gis-driver.c b/gnome-initial-setup/gis-driver.c
index 8459efa..574887c 100644
--- a/gnome-initial-setup/gis-driver.c
+++ b/gnome-initial-setup/gis-driver.c
@@ -788,7 +788,9 @@ gis_driver_startup (GApplication *app)
 
   G_APPLICATION_CLASS (gis_driver_parent_class)->startup (app);
 
+#if !WEBKIT_CHECK_VERSION(2, 39, 5)
   webkit_web_context_set_sandbox_enabled (context, TRUE);
+#endif
 
   if (driver->mode == GIS_DRIVER_MODE_NEW_USER)
     connect_to_gdm (driver);
